### ISSUE TITLE
Regex broken

### ISSUE THREAD

**Issue by gamer01** [notepadqq/notepadqq#644]
**I'm submitting a ...**
```
[x] bug report
[ ] feature request
[ ] support request
```

**Current behavior**
can't use regexes in search and replace

**Expected behavior**
regexes work ^^

**Minimal reproduction of the problem with instructions**
1. create document with content "test.."
1. open search
1. enable regex
1. try to find matches for "."
-> it will only find the . in the document, but it should match every single character.

**Please tell us about your environment:**
notepadqq 1.4; archlinux
qt5-base 5.10.1

**Comment by JuBan1** [comment]
You're right- it must've gotten broken at some point during refactoring. Will fix. In the meantime, you can use the advanced search feature (CTRL+Shift+F) as a workaround. Its regex search works properly (hopefully!).

**Comment by gamer01** [comment]
Thanks :) whats your release plan, when can i expect it to be rolled out, or should i rather build it from git if i need it fixed.

**Comment by danieleds** [comment]
A new version will be released maybe today. Package availability depends on the maintainers for your distribution.
